.github/workflows/snyk.yml

```diff
@@ -0,0 +1,22 @@
+# This action runs every day at 6 AM and on every push
+# If the branch it's running on is main then it will run snyk monitor (reports vulnerabilities to snyk.io)
+# Otherwise it will run snyk test
+name: Snyk
+
+on:
+  schedule:
+    - cron: '0 6 * * *'
+  push:
+    branches:
+      - add-snyk-workflow
+  workflow_dispatch:
+
+jobs:
+  security:
+    uses: guardian/.github/.github/workflows/sbt-node-snyk.yml@main
+    with:
+      SKIP_SBT: true
+      DEBUG: true
+      ORG: guardian-people
```

Comment:
guardian-people is an org that contains lots of off-cuts. One of the things i was talking to @Georges-GNM about was that we can make T&C a custom one if you let me know what you want it to be called! `guardian-<team-identifier>` is the typical pattern